src/DateUtils.js

```diff
@@ -27,28 +54,39 @@ module.exports = {
     formatDate: function(date) {
         // date.toLocaleTimeString is completely system dependent.
         // just go 24h for now
+        function pad(n) {
+            return (n < 10 ? '0' : '') + n;
+        }
+        const days = getDaysArray();
+        const months = getMonthsArray();
 
+		// TODO: use standard date localize function provided in counterpart
+		var hoursAndMinutes = pad(date.getHours()) + ':' + pad(date.getMinutes());
```

Comment:
indentation seems worryingly broken here - are these tabs?! if so, please please please go through and turn the tabs into 4-spaces please in accordance to the codestyle :)